cmd/node_manager_server/main.go

```diff
@@ -49,23 +49,27 @@ func installMassaNode(c *gin.Context) {
 
 	var input node_manager.InstallNodeInput
 	if err := c.ShouldBind(&input); err != nil {
+		fmt.Println(err.Error())
```

Comment:
You should add context to error. Something such as:

```suggestion
		fmt.Println(fmt.Errorf("binding: %w", err))
```